models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql

```diff
@@ -0,0 +1,66 @@
+{{ config(materialized='view', alias='solvers',
```

Comment:
one very minor nitpick: we don't need to use ```materialized``` as view here, since dbt_project file defaults to view. i've tried to somewhat stay on top of that for consistency.
@augustog can you push a quick commit to remove? then if auto-test passes again (which it should), you can go ahead and merge 👍

Reply:
In that case we should also be removing this from the other solver file on mainnet

Reply:
Done.

Reply:
it's really not a big deal, but thanks for adding it